static/js/pages/independent-expenditures.js

```diff
@@ -66,6 +66,7 @@ $(document).ready(function() {
   new tables.DataTable($table, {
     title: 'Independent expenditure',
     path: 'schedules/schedule_e',
+    query: {is_notice: 'false'},
```

Comment:
Right, I was agreeing above but got pulled away. I revised the patch so that we only touch the itemized IE view. Thanks for the catch!